setup.c

```diff
@@ -1100,9 +1100,14 @@ static int safe_directory_cb(const char *key, const char *value, void *d)
 {
 	struct safe_directory_data *data = d;
 
-	if (!value || !*value)
+	if (strcmp(key, "safe.directory"))
+		return 0;
+
+	if (!value || !*value) {
```

Comment:
Not a change in this PR but I'm curious if the intended behavior of this check is that a config file like this will ignore the preceding entries in the allow-list?

```
[safe]
    directory = somepath
    directory =
```

Reply:
Yes, this is the intended design of multi-valued Git config.